base/replicator.go

```diff
@@ -90,7 +90,7 @@ func (r *Replicator) startReplication(parameters sgreplicate.ReplicationParamete
 		parameters.ReplicationId = CreateUUID()
 	}
 
-	LogTo("Replicate", "Creating replication with parameters %+v", parameters)
+	LogToR("Replicate", "Creating replication with parameters %v", UD(parameters))
```

Comment:
What about leaving as a `%+v` in case the log redaction code is upgraded in the future to leverage that?  It seems very helpful to have the parameter names along with their values here.

Reply:
`UD(parameters)` will convert the struct into a string (with `%+v` [here](https://github.com/couchbase/sync_gateway/blob/f57ca8a70cfb506bc674ecbdde7a4233f7f5e0d6/base/redactor_userdata.go#L43)). Really the `%v` in this log should be `%s`.